### ISSUE TITLE
FR: Allow Members to request ability to accept a shared in device.

### ISSUE THREAD

**Issue by eTeitzel** [tailscale/tailscale#8128]
### What are you trying to do?

Allow MEMBERS to add devices that have been shared directly to them from other tailnodes.  Currently Members can access shared devices once they are accepted but cannot accept them because they don't have access to the admin portal.  

### How should we solve this?

allow Members to request the shared device to be added to their account. Create a landing web page for Members to request the shared device be added to their account.  Then the device can sit pending the same way it would for normal Device Approval.

OR add a new option under users names in the App to have them paste the link to the shared device in. Then it can again be added to the pending approval list.

### What is the impact of not solving this?

Members have to be elevated to an Admin level to accept a shared device.  Opening them up to lots of other functions if not monitored.

**Comment by DentonGentry** [comment]
Duplicate of https://github.com/tailscale/tailscale/issues/1865

Before we can allow non-admin users to accept sharing invites we need to rethink the controls admins have of what nodes can be shared in.
